core/extension/gdextension.cpp

```diff
@@ -286,6 +286,7 @@ void GDExtension::_register_extension_class(GDExtensionClassLibraryPtr p_library
 	const GDExtensionClassCreationInfo2 class_info2 = {
 		p_extension_funcs->is_virtual, // GDExtensionBool is_virtual;
 		p_extension_funcs->is_abstract, // GDExtensionBool is_abstract;
+		true,
```

Comment:
```suggestion
		true, // GDExtensionBool is_exposed;
```
for consistency with the rest.